pysumma/preprocessing/create_local_attribute.py

```diff
@@ -0,0 +1,237 @@
+                    lulc['vegTypeIndex'][index] = 13
+                elif lulc_df['vegTypeIndex'].loc[index] == 43:
+                    lulc['vegTypeIndex'][index] = 15
+                elif lulc_df['vegTypeIndex'].loc[index] == 51 and 52 and 53 and 54 and 72:
+                    lulc['vegTypeIndex'][index] = 16
+                elif lulc_df['vegTypeIndex'].loc[index] == 31 and 62:
+                    lulc['vegTypeIndex'][index] = 17
+                elif lulc_df['vegTypeIndex'].loc[index] == 61:
+                    lulc['vegTypeIndex'][index] = 18
+                elif lulc_df['vegTypeIndex'].loc[index] == 82:
+                    lulc['vegTypeIndex'][index] = 20
+                elif lulc_df['vegTypeIndex'].loc[index] == 81:
+                    lulc['vegTypeIndex'][index] = 21
+                elif lulc_df['vegTypeIndex'].loc[index] == 84:
+                    lulc['vegTypeIndex'][index] = 22
+                elif lulc_df['vegTypeIndex'].loc[index] == 71 and 74 and 75 and 76 and 77 and 83:
+                    lulc['vegTypeIndex'][index] = 23
+                elif lulc_df['vegTypeIndex'].loc[index] == 91 and 92:
+                    lulc['vegTypeIndex'][index] = 24
+                elif lulc_df['vegTypeIndex'].loc[index] == 73:
+                    lulc['vegTypeIndex'][index] = 27
+                else:
+                    pass
+
+        elif option == "MODIFIED_IGBP_MODIS_NOAH":
+            for index, raster in lulc_df.iterrows():
+                if lulc_df['vegTypeIndex'].loc[index] == 42:
+                    lulc['vegTypeIndex'][index] = 2
+                elif lulc_df['vegTypeIndex'].loc[index] == 41:
+                    lulc['vegTypeIndex'][index] = 4
+                elif lulc_df['vegTypeIndex'].loc[index] == 43:
+                    lulc['vegTypeIndex'][index] = 5
+                elif lulc_df['vegTypeIndex'].loc[index] == 61 and 62:
+                    lulc['vegTypeIndex'][index] = 11
+                elif lulc_df['vegTypeIndex'].loc[index] == 21 and 22:
+                    lulc['vegTypeIndex'][index] = 12
+                elif lulc_df['vegTypeIndex'].loc[index] == 11 and 12 and 13 and 14 and 16 and 17:
+                    lulc['vegTypeIndex'][index] = 13
+                elif lulc_df['vegTypeIndex'].loc[index] == 23 and 24 and 31 and 32 and 33:
+                    lulc['vegTypeIndex'][index] = 14
+                elif lulc_df['vegTypeIndex'].loc[index] == 91 and 92:
+                    lulc['vegTypeIndex'][index] = 15
+                elif lulc_df['vegTypeIndex'].loc[index] == 51 and 52 and 53 and 54:
+                    lulc['vegTypeIndex'][index] = 17
+                elif lulc_df['vegTypeIndex'].loc[index] == 84:
+                    lulc['vegTypeIndex'][index] = 19
+                elif lulc_df['vegTypeIndex'].loc[index] == 71 and 72 and 73 and 74 and 75 and 76 and 77 and 81 and 82 and 83:
+                    lulc['vegTypeIndex'][index] = 20
+                else:
+                    pass
+        return lulc
```

Comment:
As above, just having these big chains of `elif`s on raw integers is very mysterious. I have no idea what each of these stands for.